### PR TITLE
added chapter for Meteor cloud radar

### PR DESCRIPTION
**Pull request by d70-t**

### CODE REVIEW

File: how_to_eurec4a/meteor_cloudradar.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Intake will this do this by default.
+Let's obtain the EUREC4A intake catalog:
+
+```{code-cell} ipython3
+import eurec4a
+cat = eurec4a.get_intake_catalog()
+```
+
+## available products
+The the `LIMRAD94` cloud radar offers multiple products which can be accessed using names and additional parameters.
```
Comment (jroettenbacher) [inline how_to_eurec4a/meteor_cloudradar.md:26]: one 'the' too many

File: how_to_eurec4a/meteor_cloudradar.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Let's see which products and parameters available for the cloud radar.
```
Comment (jroettenbacher) [inline how_to_eurec4a/meteor_cloudradar.md:27]: are available

File: how_to_eurec4a/meteor_cloudradar.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.7.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# reading cloud radar data
```
Comment (jroettenbacher) [inline how_to_eurec4a/meteor_cloudradar.md:14]: I would start headings with a captial letter

**Review by jroettenbacher — COMMENTED** [review]
Just some minor comments. Other than that, this looks great! Thanks for that

**Comment by d70-t** [comment]
@jroettenbacher, thank you for reviewing this. I've implemented all your suggestions. Should we proceed merging it?

**Pull request merged**